pkg/asset/quota/aws/aws.go

```diff
@@ -131,13 +133,21 @@ func others() []quota.Constraint {
 func machineTypeToQuota(t string, instanceTypes map[string]InstanceTypeInfo) quota.Constraint {
 	info, ok := instanceTypes[t]
 	if !ok {
-		return quota.Constraint{Name: "ec2/L-7295265B", Count: 0}
+		return quota.Constraint{Name: "unknown", Count: 0}
```

Comment:
I think that this needs to stay as a valid quota label. The installer is going to check this quota but check for enough quota for 0 cpus. That is unless there is conditional somewhere to skip the quota check when the count is 0.